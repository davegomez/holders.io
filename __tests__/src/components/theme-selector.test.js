import React from 'react'
import ThemeSelector from '../../../src/components/theme-selector'
import renderer from 'react-test-renderer'
import { indetity } from '../test-utils'

test('ThemeSelector component', () => {
  const component = renderer.create(
    <ThemeSelector onClick={ indetity } />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
