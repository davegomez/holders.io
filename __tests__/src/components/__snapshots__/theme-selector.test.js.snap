// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeSelector component 1`] = `
<div
  data-jsx={2110134670}
>
  <button
    className="active"
    data-jsx={43889984}
    data-theme="empty"
    onClick={[Function]}
    style={null}
  />
  <button
    className={null}
    data-jsx={43889984}
    data-theme="puppy"
    onClick={[Function]}
    style={
      Object {
        "backgroundImage": "url('/static/img/themes/puppy.jpg')",
      }
    }
  />
  <button
    className={null}
    data-jsx={43889984}
    data-theme="kitten"
    onClick={[Function]}
    style={
      Object {
        "backgroundImage": "url('/static/img/themes/kitten.jpg')",
      }
    }
  />
</div>
`;
